sdk/digitaltwins/Azure.DigitalTwins.Core/README.md

```diff
@@ -6,26 +6,38 @@ This library provides access to the Azure Digital Twins service for managing twi
 
 ## Getting started
 
-### Authenticate the Client
+The complete Microsoft Azure SDK can be downloaded from the [Microsoft Azure Downloads][[microsoft_sdk_download]] Page and ships with support for building deployment packages, integrating with tooling, rich command line tooling, and more.
```

Comment:
Is microsoft_sdk_download supposed to be in double square brackets?

Reply:
no !